nixos/doc/manual/development/releases.xml

```diff
@@ -8,359 +8,9 @@
   <title>Release process</title>
 
   <para>
-   Going through an example of releasing NixOS 19.09:
+   This section has been moved to <link xlink:href="https://github.com/NixOS/release-wiki">the release wiki</link>.
```

Comment:
It doesn't make sense to link it to the wiki like this. Instead there needs to be a section explaining the how and why of the release process, and then say "If you're interested in how a NixOS release actually happens please see https://nixos.github.io/release-wiki/` and that's the rendered link. Speaking of which the github action is failing there.

Reply:
I wouldn't really expect for the release process to be there in the first place. I think this mostly benefits people who are used to it being there. 

New users will probably find it confusing that it's mentioned at all.